lib/dom.js

```diff
@@ -3568,6 +3568,36 @@ declare class ClientRectList { // extension
   [index: number]: ClientRect;
 }
 
+declare class DOMRectReadOnly {
```

Comment:
missing the constructor: https://drafts.fxtf.org/geometry/#dom-domrectreadonly-domrectreadonly